pubspec.yaml

```diff
@@ -1,7 +1,8 @@
 name: clevertap_plugin
 description: The CleverTap Flutter SDK for Mobile Customer Engagement,Analytics and Retention solutions.
-version: 1.2.3
+version: 1.3.0
```

Comment:
are we taking the latest clevertap sdk?

Reply:
Han this was suppose to be in the flutterUpgrade branch itself.